src/cml.js

```diff
@@ -464,7 +465,7 @@ class CML {
       await exec(`git checkout -b ${source}`);
       await exec(`git add ${paths.join(' ')}`);
       let commitMessage = `CML PR for ${shaShort}`;
-      if (!(merge || rebase || squash)) {
+      if ( skipCI ||  !(merge || rebase || squash)){
```

Comment:
```suggestion
      if (skipCI || !(merge || rebase || squash)){
```


but as per the last line in [this comment](https://github.com/iterative/cml/issues/1023#issuecomment-1140154009) I wonder if `skipCI` should mean ` [skip ci]` is added to `title` rather than `commitMessage`.